lib/Service/SampleConversationsService.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+/**
+ * SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Talk\Service;
+
+use OCA\Talk\Chat\ChatManager;
+use OCA\Talk\Exceptions\RoomNotFoundException;
+use OCA\Talk\Manager;
+use OCA\Talk\Model\Attendee;
+use OCA\Talk\Room;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\Files\IRootFolder;
+use OCP\Files\NotFoundException;
+use OCP\Files\NotPermittedException;
+use OCP\IConfig;
+use OCP\IL10N;
+use OCP\IURLGenerator;
+use OCP\IUser;
+use OCP\IUserManager;
+use OCP\PreConditionNotMetException;
+use OCP\Security\ISecureRandom;
+
+class SampleConversationsService {
+	public function __construct(
+		protected IConfig $config,
+		protected IUserManager $userManager,
+		protected Manager $manager,
+		protected ChatManager $chatManager,
+		protected RoomService $roomService,
+		protected AvatarService $avatarService,
+		protected ParticipantService $participantService,
+		protected ISecureRandom $secureRandom,
+		protected IRootFolder $rootFolder,
+		protected IURLGenerator $url,
+		protected ITimeFactory $timeFactory,
+		protected IL10N $l,
+	) {
+	}
+
+	public function initialCreateSamples(string $userId): void {
+		$created = $this->config->getUserValue($userId, 'spreed', 'samples_created');
+		if ($created !== '') {
+			return;
+		}
+
+		$this->config->setUserValue($userId, 'spreed', 'samples_created', $this->timeFactory->now()->format(\DateTime::ATOM));
+
+		$user = $this->userManager->get($userId);
+		if (!$user instanceof IUser) {
+			throw new \InvalidArgumentException('User not found');
+		}
+
+		$this->defaultSampleConversation($user);
+	}
+
+	protected function defaultSampleConversation(IUser $user): void {
+		$userFolder = $this->rootFolder->getUserFolder($user->getUID());
+		$room = $this->roomService->createConversation(
+			Room::TYPE_GROUP,
+			$this->l->t('Let´s get started!'),
+			$user,
+			'sample',
+			$user->getUID()
+		);
+
+		$this->avatarService->setAvatarFromEmoji($room, '💡', '0082c9');
```

Comment:
The default experience is quite blue, should we use another color?
![grafik](https://github.com/user-attachments/assets/bc1f377a-328d-48c6-aa4a-cd299f38c828)